### DISCUSSION TITLE
i installed pterotactyl on aapanel

### DISCUSSION THREAD

**Discussion by 880099** [pterodactyl/panel#4045] [Support] [answered]
How to solve the problem?
![image](https://user-images.githubusercontent.com/90360840/163228914-5f33d381-c928-4890-b703-225239f48252.png)

**Comment by 880099** [comment]
![image](https://user-images.githubusercontent.com/90360840/163230419-08b43c33-7454-4e81-8ae0-e06e7b93258d.png)

**Comment by 880099** [comment]
im installed the Pterodactyl on [aapanel](www.aapanel.com)l,Many errors have occurred.

**Comment by jqshuv** [comment, marked as answer]
The app is trying to bind to the ports 55001 and 2020 but they're currently used by another app. You have to stop the other app or change the ports.
